assets/queries/terraform/aws/auto_scaling_group_with_no_associated_elb/query.rego

```diff
@@ -74,3 +80,13 @@ CxPolicy[result] {
 		"searchLine": common_lib.build_search_line(["module", name, "load_balancers"], []),
 	}
 }
+
+has_target_group_arns(resource, key){
+	common_lib.valid_key(resource, key)
+	not is_array(resource[key])
+	resource[key] != ""
+} else{
+	common_lib.valid_key(resource, key)
+	is_array(resource[key])
+	count(resource[key]) > 0
+}
```

Comment:
I think `common_lib.valid_key(resource, key)` is unnecessary since is_array `is_array(resource[key])` will only be true, if key exists in resource, for example.

```suggestion
has_target_group_arns(resource, key){
	not is_array(resource[key])
	resource[key] != ""
} else{
	is_array(resource[key])
	count(resource[key]) > 0
}
```